src/components/transport_manager/docs/SDL.SDD.WebEngineSupport.dox

```diff
@@ -0,0 +1,127 @@
+/**
+\page web_engine_support Web Engine Support Detailed Design
+## Table of contents
+- \subpage wes_intoduction
+  + \ref wes_rationale                "1.1 Rationale"
+  + \ref wes_scope                    "1.2 Scope"
+- \subpage wes_detail_design
+  + \ref wes_design_solutions         "2.1 Design solutions"
+  + \ref wes_class_structure          "2.2 Class Structure"
+  + \ref wes_sequence_diagram         "2.3 Sequence diagram"
+- \subpage wes_data_structures
+  + \ref wes_data_structure        "3.1 Data structures"
+- \subpage wes_references_and_history
+  + \ref wes_history                  "4.1 References"
+*/
+//-----------------------------------------------------------
+/**
+\page wes_intoduction 1 Introduction
+The document is intended to support software developers, 
+maintenance and integration engineers with sufficient,
+detailed information concerning the design, development and
+deployment concepts, to accomplish their respective tasks without reliance on the authors.
+
+\anchor wes_rationale
+## 1.1 Rationale
+Boost asio libraries were utilized to enable websocket server connection for web engine.
+To split responsibilities, the following entities were introduces to enable server connections:
+- WebSocketListener
+- WebSocketConnection
+- WebSocketSession
+- WebSocketSecureSession
+
+
+\anchor wes_scope
+## 1.2 Scope
+All aforementioned entities are part of transport_manager namespace
+
+*/
+//-----------------------------------------------------------
+/**
+\page wes_detail_design 2 Component detail design
+\anchor wes_design_solutions
+### 2.1 Design solutions
+
+- Introducing extra layers to pass relevant websocket connection data between them
```

Comment:
Can you specify what `them` is?

Reply:
rewrote this point in 3af95076c6c05d68e46c9d08a5800b9cdb8536df